benchmarks/index.md

```diff
@@ -64,66 +49,3 @@ benchmark_height_mobile: 6000
 <p>
     To learn more about OpenSearch performance improvements through version 2.14, read <a href="https://opensearch.org/blog/opensearch-performance-2.14/">this blog post</a>.
```

Comment:
This link should dynamically use the current version.

Reply:
This points to the last published performance blog. We are in the process of publishing a new blog, will updated once it is up.